### PR TITLE
Pre requirements to use this vscode extension

### PR DESCRIPTION
**Pull request by RafaelGorski**
Hi,

I faced a question mark when I start to use this extension because I suppose that all dependencies was setup. However this project only wrap the yo CLI inside the vscode. So I suggest this change to include a topic to explain the pre-requirements to use it to new users which are not familiar with this ecosystem of tools.

Thank you,
Rafael

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ generators will pop-up. The default generator is marked by a ★.
 
 ![](https://github.com/SamVerschueren/vscode-yo/raw/master/media/sub-generator.gif)
 
+### Pre-requirements
+* [Node] (https://nodejs.org)
```
Comment (sindresorhus) [inline README.md]: Node.js

File: README.md [hunk 2]
```diff
@@ -16,6 +16,11 @@ generators will pop-up. The default generator is marked by a ★.
...
+* [NPM] (https://www.npmjs.com)
```
Comment (sindresorhus) [inline README.md]: npm

File: README.md [hunk 3]
```diff
@@ -16,6 +16,11 @@ generators will pop-up. The default generator is marked by a ★.
 
 ![](https://github.com/SamVerschueren/vscode-yo/raw/master/media/sub-generator.gif)
 
+### Pre-requirements
+* [Node] (https://nodejs.org)
+* [NPM] (https://www.npmjs.com) 
+* [Yeoman] (http://yeoman.io)
+* An generator from your choice [Yeoman generators](http://yeoman.io/generators/)
```
Comment (sindresorhus) [inline README.md]:
```
A [Yeoman generator](http://yeoman.io/generators/) of choice
```

**Comment by RafaelGorski** [comment]
Cool

**Comment by SamVerschueren** [comment]
Can you make the changes? Haven't merged it yet.

**Comment by RafaelGorski** [comment]
@SamVerschueren done

**Comment by SamVerschueren** [comment]
@RafaelGorski thanks!

**Pull request merged**